dart/optimizer/Problem.h

```diff
@@ -71,6 +73,28 @@ class Problem
   /// \brief Set initial guess for opimization parameters
   const Eigen::VectorXd& getInitialGuess() const;
 
+  /// \brief Add a seed for the Solver to use as a hint for the neighborhood of
+  /// the solution.
+  void addSeed(const Eigen::VectorXd& _seed);
+
+  /// \brief Get a mutable reference of the seed for the specified index. If an
+  /// out-of-bounds index is provided a warning will print, and a reference to
+  /// the initial guess will be returned instead.
+  Eigen::VectorXd& getSeed(size_t _index);
+
+  /// \brief An immutable version of getSeed(size_t)
+  const Eigen::VectorXd& getSeed(size_t _index) const;
+
+  /// \brief Get a mutable reference to the full vector of seeds that this
+  /// Problem currently contains
+  std::vector<Eigen::VectorXd>& getAllSeeds();
```

Comment:
We've used `get[Name]s` for the functions that return all the entries (e.g., `getBodyNodes()`). Could we make this function name to be consist with them as `getSeeds()`? Or, do we need to consider to use 'All' for this kind of functions?